services/graph/pkg/errorcode/cs3.go

```diff
@@ -45,7 +45,7 @@ func FromCS3Status(status *cs3rpc.Status, inerr error, ignore ...cs3rpc.Code) *E
 	case code == cs3rpc.Code_CODE_ALREADY_EXISTS:
 		err.errorCode = NameAlreadyExists
 	case code == cs3rpc.Code_CODE_FAILED_PRECONDITION:
-		err.errorCode = PreconditionFailed
+		err.errorCode = InvalidRequest
```

Comment:
@rhafer I needed to fix that. We had misunderstandings about that CS3 Status already in the past.